stdlib/public/runtime/ImageInspectionELF.cpp

```diff
@@ -144,6 +152,8 @@ int swift::lookupSymbol(const void *address, SymbolInfo *info) {
   return 1;
 }
 
+#endif
```

Comment:
Why not just use the alternate version always?  The cost is nearly negligible - the cost is largely associated with the fact that the information from the ELF header parsing *may* be cached by the loader, which is **not** guaranteed.